docs/diagrams/DeleteDateActivityDiagram.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+skinparam defaultTextAlignment center
+
+start
+:User requests to\ndelete cases from a given date;
+:Check that indexes and dates\nhave not been used together;
```

Comment:
```suggestion
:Check that other prefixes \nhave not been used together;
```
No reason to treat indices differently from date ranges etc
or can remove entirely since you say user is just deleting dates